packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -79,7 +79,31 @@ Okta will derive client request context directly from the HTTP request headers a
 
 Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version such as `Oktaprise/1.1`.
 
-> If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+### Format a User-Agent string for native mobile apps
+
+Make sure that the User-Agent string that your app constructs is in the correct format. This ensures that Okta can parse the OS and Browser fields. Good User-Agent strings that can be correctly parsed by Okta contain browser information, system information, platform, platform details, and any extensions.
+
+We recommend that you use a template like the following to format the User-Agent string:
+
+`User-Agent: Mozilla/5.0 (<system-information>) <platform> (<platform-details>) <extensions>`
+
+Alternatively, you can make sure that the following string values are present in the User-Agent string so that the OS and browser are detected:
+
+**Any Device Operating System**
+
+If you have a browser string such as `chrome` or `safari`, add that to the User-Agent string.
+
+**iOS**
+
+Include the words `apple` or `ios` and at least one of these values: `iphone`, `ipad`, `ipod`, `ipad`.
+
+**Android**
+
+Include the words `android` or `samsung`, which infers that Android is the operating system.
+
+> **Note:** Case doesn't matter since Okta converts the string to lowercase before matching.
```

Comment:
We could also add a note about testing to make sure that `OS` and `Browser` show up properly in the system log events.

Reply:
@srinivasanagandla-okta Can you help me out with the wording of this note?

Reply:
@srinivasanagandla-okta - @lilyhsiao-okta has the following comment about the note. Can I get a little more information for this note? Such as a place to direct the user for more information on how to test? Here is Lily's comment:

How would the developer know how to test? if we include this Note we should also describe or point to info on how to test. For ex, "to test if Okta can parse the string, check the System log of the request that was passed. See the [System Log] ... "